File: tracsmobile/src/main/java/edu/txstate/mobile/tracs/util/SettingsListView.java
package edu.txstate.mobile.tracs.util;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.ExpandableListView;

public class SettingsListView extends ExpandableListView {
    private static final String TAG = "SettingsListView";

    public SettingsListView(Context context) {
        super(context);
    }

    public SettingsListView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public SettingsListView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }
}

File: tracsmobile/src/main/java/edu/txstate/mobile/tracs/util/SettingsListView.kt
package edu.txstate.mobile.tracs.util

import android.content.Context
import android.util.AttributeSet
import android.widget.ExpandableListView

class SettingsListView : ExpandableListView {

    companion object {
        val TAG = "SettingsListView"
    }

    constructor(context: Context): super(context)
    constructor(context: Context, attrs: AttributeSet): super(context, attrs)
    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int): super(context, attrs, defStyleAttr)

}